receiver/k8sclusterreceiver/metadata.yaml

```diff
@@ -258,6 +258,46 @@ resource_attributes:
     type: string
     enabled: true
 
+  k8s.ingress.uid:
+    description: The UID of the Role.
```

Comment:
This should be `The UID of the Ingress`. Same for other metadata below.

Reply:
I see, Updated the metadata